model/job-management/src/main/proto/beam_job_api.proto

```diff
@@ -176,3 +179,34 @@ message JobState {
     CANCELLING = 10;
   }
 }
+
+
+// DescribePipelineOptions provides metadata about the options supported by a runner.
+// It will be used by the SDK client to validate the options specified by or
+// list available options to the user. // Throws error GRPC_STATUS_UNAVAILABLE if server is down
+message DescribePipelineOptionsRequest {
+  // TODO: base options to support proxy functionality
```

Comment:
What do you mean by that?

Reply:
For example, if the runner is an intermediate, then it may require additional information to figure out how to route this request.

Reply:
I see, thanks for clarifying.